conversions/humidity.js

```diff
@@ -0,0 +1,36 @@
+
+let tempMessage = (hum, inst, src) => {
+    return [{
+      pgn: 130313,
+      SID: 0xff,
+      "Instance": inst,
+      "Source": src,
+      "Actual Humidity": hum,
+    }]
+  }
+  
+  module.exports = (app, plugin) => {
+    return [{
+      pgn: 130313,
+      title: 'Outside Humidity (130313)',
+      optionKey: 'HUMIDITY_OUTSIDE',
+      keys: [
+        "environment.outside.humidity"
```

Comment:
I believe these should be `relativeHumidity`

https://signalk.org/specification/1.7.0/doc/vesselsBranch.html#vesselsregexpenvironmentoutsidehumidity